src/facade/dragonfly_connection.cc

```diff
@@ -654,6 +654,7 @@ void Connection::HandleRequests() {
   if (http_res) {
     cc_.reset(service_->CreateContext(peer, this));
     if (*http_res) {
+      stats_ = &tl_facade_stats->conn_stats;
```

Comment:
sanity

Reply:
let's move it to line 609 and remove `stats_ = &tl_facade_stats->conn_stats;` from ConnectionFlow()